src/overlays/actors/ovl_Demo_Moonend/z_demo_moonend.h

```diff
@@ -2,14 +2,21 @@
 #define Z_DEMO_MOONEND_H
 
 #include "global.h"
+#include "objects/object_moonend/object_moonend.h"
```

Comment:
You include `"objects/object_moonend/object_moonend.h"` in both the `.c` and the `.h` file, probably makes sense to just include it in one of them. And more of a question to other contributors (outside the scope of this PR) but do we want these includes in the header or directly in the c file? It's currently very inconsistent

Reply:
Got discussed in discord, and espcially since this will need the limb enum eventually the include should be in the `.h` file. Could you remove the one from the `.c` one?